drafts/securing-openliberty-with-openidconnect-and-jwt.adoc

```diff
@@ -11,49 +11,42 @@ blog_description: "A look at the OpenID Connect, SAML and Json Web Token securit
 = Securing Open Liberty apps and microservices with OpenID Connect and JWT
 Bruce Tiffany <https://github.com/brutif>
 
-Open Liberty now includes more features for securing apps with external security providers.  
-Let's take a quick look at why and how to use some of these features, focusing on using OpenID Connect to secure microservices.
+Open Liberty now includes more open source security features to enable you to secure your apps with external security providers. Let's take a quick look at why and how to use some of these features.
 
-== Concepts:
-* SAML and OpenID Connect are standards-based technologies for using an external security provider to secure your applications. 
-+
-* Json Web Tokens (JWT) are identity credentials with a standardized format that can be exchanged between processes. They're secured against tampering by use of digital signatures.
+== Authenticating microservices on Open Liberty
```

Comment:
We authenticate users, we secure microservices.  Thinking about how to reword this.